packages/zero-runtime/package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@mui/zero-runtime",
-  "version": "0.0.1-alpha.0",
+  "version": "0.0.1-alpha.1",
```

```suggestion
  "version": "0.0.1-alpha.0",
```

Shouldn't this be zero? It's the first version.

---

Same for the other package

---

Good point.

---

Next time, we could consider dropping the `alpha` and starting with `0.1.0` according to [semver](https://semver.org/#how-should-i-deal-with-revisions-in-the-0yz-initial-development-phase).